test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2516,8 +2516,8 @@ def forward(self, x):
                 arr = x.split([3, 4, 1, 1, 2, 3, 2], 0)
                 res2 = torch.zeros(3, 4, dtype=torch.long)
                 for i in range(len(arr)):
-                    res = res.append(arr[i].sum(0, False))
-                    res1 = res1.append(arr[-1 - i].sum(0, False))
+                    res = res + [arr[i].sum(0, False)]
```

Comment:
is the add between tensor list sort of a workaround for the issue of of empty list that is populated inside the loop via append?

Reply:
The purpose was not to workaround, this happens to be another way of working with list of tensors aside from append, and has to be supported as well. The issue with append will be fixed in another PR.

Reply:
Then why are we replacing the tests here? Can you add the new tests without removing old ones?

Reply:
Good point, restored prev tests and added new one.